src/CelesteTypes.jl

```diff
@@ -193,34 +205,14 @@ end
 # A vector of variational parameters.  The outer index is
 # of celestial objects, and the inner index is over individual
 # parameters for that object (referenced using ParamIndex).
+
 # TODO: use a matrix here, in conjunction with ArrayViews.jl (?)
+# TODO: Julia noob question -- is there a way to enforce the
+# differences between these variable types?  For now, this is
+# just helpful notation.
```

Comment:
I was wondering that too. I suspect not, because if VariationalParams wasn't just another name for Vector{Vector{Float64}}, we couldn't index into it, multiply it, etc. using functions designed for vectors.